addons/payment/wizards/payment_acquirer_onboarding_wizard.py

```diff
@@ -117,11 +123,19 @@ def add_payment_methods(self):
             new_env = api.Environment(self.env.cr, self.env.uid, self.env.context)
 
             if self.payment_method == 'paypal':
-                acquirer = new_env.ref('payment.payment_acquirer_paypal', raise_if_not_found=False)
+                acquirer_count = new_env['payment.acquirer'].search_count(
+                    [('company_id', '=', self.env.company.id), ('name', '=', 'PayPal')]
+                )
+                if acquirer_count == 0:
+                    new_acquirer = self.env.ref('payment.payment_acquirer_paypal').with_company(self.env.company.id).sudo().copy()
+                    new_acquirer.company_id = self.env.company.id
+                acquirer = new_env['payment.acquirer'].search(
+                    [('company_id', '=', self.env.company.id), ('name', '=', 'PayPal')]
+                )
                 default_journal = new_env['account.journal'].search(
                     [('type', '=', 'bank'), ('company_id', '=', new_env.company.id)], limit=1
                 )
-                new_env.ref('payment.payment_acquirer_paypal').write({
+                new_env['payment.acquirer'].search([('company_id', '=', self.env.company.id), ('name', '=', 'PayPal')], limit=1).write({
```

Comment:
Looks like it's the same as the one a few lines before 
```suggestion
                acquirer.write({
```